extensions/ql-vscode/src/cli.ts

```diff
@@ -929,6 +930,12 @@ export class CodeQLCliServer implements Disposable {
         throw new Error('No distribution found');
     }
   }
+
+  private getAdditionalPacksArg(paths: string[]): string[] {
```

Comment:
Minor: Is `Args` more appropriate here since the return type is an array?

Reply:
I used `Arg` (singular) since this method returns the `--additional-packs` argument.